backend/organization/admin.py

```diff
@@ -28,3 +28,19 @@ class ProjectAdmin(admin.ModelAdmin):
 
 
 admin.site.register(Project, ProjectAdmin)
+
+
+class ProjectMemberAdmin(admin.ModelAdmin):
+    search_fields = ('user__name', 'user__id', 'project__name',)
+    list_filter = ('role__name',)
+
+
+admin.site.register(ProjectMember, ProjectMemberAdmin)
```

Comment:
Does this mean, that they can do something through the django admin interface/api directly?

Reply:
No it is just for us to manage these tables.